engine/src/test/scala/cromwell/engine/WorkflowDescriptorSpec.scala

```diff
@@ -171,7 +171,9 @@ class WorkflowDescriptorSpec extends FlatSpec with Matchers with ScalaFutures {
           1,
           Map.empty,
           None,
-          None)
+          None,
+          false,
+          false)
         call -> Seq(metadata)
     }).toMap
 
```

Comment:
There doesn't seem to be a substantive test that these fields are correctly filled out for hits / non-hits and ALLOW_RESULTS_REUSE turned on or not.